packages/storybook/config/main.ts

```diff
@@ -4,12 +4,14 @@ const config: StorybookConfig = {
   stories: ['../src/**/*stories.@(js|jsx|ts|tsx)', '../src/**/*.mdx'],
   addons: [
     '@storybook/addon-a11y',
+    '@storybook/addon-backgrounds',
     '@storybook/addon-controls',
     '@storybook/addon-docs',
     '@storybook/addon-viewport',
     '@storybook/addon-actions',
     '@storybook/addon-interactions',
     '@storybook/addon-links',
+    '@chromatic-com/storybook',
```

Comment:
Deze heb je alleen nodig als je _in_ Storybook zelf wat met Chromatic zou willen. Het kan geen kwaad hem te laten staan.

Reply:
Heb m weggehaald. Als we m nodig hebben voegen we m wel weer toe